site/content/en/contributions/RELEASING.md

```diff
@@ -111,57 +112,50 @@ export GITHUB_REMOTE=origin
          make docs-release TAG=v${MAJOR_VERSION}.${MINOR_VERSION}.0
       ```
 
-   1. Update the `Documentation` referred link on the menu in `site/hugo.toml`:
-
-      **DON'T FORGOT TO MOVE IT UNDER `LATEST`**
-
-      ```shell
-      [[menu.main]]
-         name = "Documentation"
-         weight = -101
-         pre = "<i class='fas fa-book pr-2'></i>"
-         url = "/v1.1"
-      ```
-
-   1. Update `site/layouts/shortcodes/helm-version.html` base on latest minor version.
+   1. Update `site/layouts/shortcodes/helm-version.html`, add the latest version of the minor release, and update the short code for `{{- with (strings.HasPrefix $pagePrefix "doc") -}}` to the latest minor version.
```

Comment:
the index number seems not right, can you also update them?

Reply:
You can always use 1 for ordered list, it's a Markdown standard. https://www.markdownguide.org/basic-syntax/#ordered-lists